tests/rustworkx_tests/digraph/test_nodes.py

```diff
@@ -194,6 +194,34 @@ def test_topo_sort_with_cycle(self):
         dag.add_edge(node_b, node_a, {})
         self.assertRaises(rustworkx.DAGHasCycle, rustworkx.topological_sort, dag)
 
+    def test_topo_generations_empty(self):
```

Comment:
And maybe include a graph that is not a DAG to trigger the condition e.g. `1 <-> 2 -> 3`

Reply:
I think `test_topo_generations_has_cycle` has already covered this?